Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/Material/MaterialComponentBus.h

```diff
@@ -41,12 +41,56 @@ namespace AZ
             virtual const AZ::Data::AssetId GetMaterialOverride(const MaterialAssignmentId& materialAssignmentId) const = 0;
             //! Clear material override
             virtual void ClearMaterialOverride(const MaterialAssignmentId& materialAssignmentId) = 0;
-            //! Set a material property value override
-            virtual void SetPropertyOverride(const MaterialAssignmentId& materialAssignmentId, const Name& propertyName, const AZStd::any& propertyValue) = 0;
-            //! Get a material property value override
-            virtual AZStd::any GetPropertyOverride(const MaterialAssignmentId& materialAssignmentId, const Name& propertyName) const = 0;
+            //! Set a material property override value wrapped by an AZStd::any
```

Comment:
will this be used elsewhere? E.g. shader canvas? Maybe makes sense to decouple into separate utility bus/class later.

Reply:
Which part are you referring to? I considered moving all of these functions/bindings to MaterialPropertyValue. I decided against it because that means there would be another intermediate node every time you wanted to get or set a value. BUT that might simplify the component API and get rid of some of the need for std any.